### ISSUE TITLE
Går det att lägga in blocket Varukorg från butiken i vänsterspalten?

### ISSUE THREAD

**Issue by recraft-ou** [alternativ-nu/retrosearch#2]
- Hur svårt är det att använda PS? Så man inte lägger ner onödiga timmar om det inte ska användas ändå sedan?
- Skulle det gå att använda bara varukorgen och kassan från PS, så man bara använde retrosearch-lösningen för att visa tidningar?
- Hur svårt är det att bygga egen enkel varukorg baserad på t ex formmail?
- Har swish, payson etc färdiga enkla kassor som…

**Comment by recraft-ou** [comment]
Det verkar gå fint att använda PS, det tar några extra sekunder att ladda in produktdatan men API:et verkar fungera bra.

**Comment by recraft-ou** [comment]
Rättigheterna i API:et för användaren kan snävas ned ännu mera. Återkommer med detaljer.

**Comment by TidningenAter** [comment]
Är detta vår strategi nu, att som första steg använda varukorgen i PS och lägga artikelsöket som antingen en egen sida eller som framsida i PS-butiken?

**Comment by recraft-ou** [comment]
Ja, jag ska få till lite bättre knappar för att lägga till i varukorgen och att gå till kassan tänkte jag, vad tror du om det?

**Comment by TidningenAter** [comment]
Skulle det vara möjligt/en bra idé att lägga in blocket Varukorg från butiken i vänsterspalten?
Då är det egentligen bara kassan i butiken kvar som behövs.

**Comment by TidningenAter** [comment]
Jag testade nu att lägga in söket i en iframe på en av cms-sidorna: http://www.alternativ.nu/butik/info/om-ater
Och det funkar ju, men är lite fult. Ramen blev för låg, vilket förstås kan fixas men det blir också konstigt med höger och vänsterspalt tror jag. Men kolla, kan det vara något att jobba på så går det att lägga det på första sidan i butiken. 

Tyvärr uppdateras inte cart-blocket inte "live" när man klickar köp, som det gör när man klickar på en "riktig" köp-knapp i butiken, utan man får ladda om sidan för att blocket ska uppdateras, men det är väl för att den ligger i en iframe.
